### DISCUSSION TITLE
Hyper V instances

### DISCUSSION THREAD

**Discussion by johnjohnk2** [microsoft/WSL#7180] [Q&A]
Hi,
I d like to understand why I have two Hyper-V instances. I think i did a bad attempt when installing Arch Linux on WSL2 and I have now two Hyper V instances. How could I remedy that ? I can't just delete the first instance because it is created again at reboot. 
Thanks for helping 

![image](https://user-images.githubusercontent.com/36307676/125700575-c123bc73-10d6-4a30-b10c-d253c50b9019.png)

**Comment by Biswa96** [comment]
Those are not two Hyper-V instances. Those are two virtual ethernet network switch - one is `vEthernet (Default Switch)` created by default for any Hyper-V VMs, another is `vEthernet (WSL)` created for WSL2 VMs specifically.

**Comment by johnjohnk2** [comment]
@Biswa96 Thanks a lot, so it is perfectly normal ?

**Reply by Biswa96** [reply]
Yup.

**Reply by johnjohnk2** [reply]
@Biswa96 How can I check the first Hyper V VM corresponding to which VM? Thanks

**Reply by johnjohnk2** [reply]
@Lite5h4dow ( for following the post which could be interesting if any similar problem is asked on Discord)

**Reply by Biswa96** [reply]
Apology, I can't understand the first question. What do you mean by "first Hyper-V VM"?

**Reply by johnjohnk2** [reply]
Sorry i meant the first ethernet network switch

**Reply by Biswa96** [reply]
Run `ipconfig /all`

**Reply by johnjohnk2** [reply]
OK i have this 
![image](https://user-images.githubusercontent.com/36307676/126234208-0dd35498-3a8f-4a37-965f-7329351474e8.png)

**Reply by Biswa96** [reply]
The Description of the interfaces are same as in Device Manager.

**Reply by johnjohnk2** [reply]
ok ok so still perfectly normal ?
